src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -1660,6 +1666,13 @@ bool Eltwise::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op
                 return false;
             }
         }
+        if (const auto select = std::dynamic_pointer_cast<const ov::op::v1::Select>(op)) {
```

Comment:
mgraph RTTI is more preferred to use with ngraph types.
```suggestion
        if (const auto select = ov::as_type_ptr<const ov::op::v1::Select>(op)) {
```

Reply:
OK, I replaced dynamic cast with ngraph RTTI in eltwise.cpp for ngraph types